### ISSUE TITLE
Any possible interaction with the org-babel literate programming?

### ISSUE THREAD

**Issue by falematte** [fredrikekre/Literate.jl#231]

**Comment by fredrikekre** [comment]
Are you talking about https://orgmode.org/worg/org-contrib/babel/ ?
If there are changes in Literate necessary to accomodate this it would be good to include in the issue here. I don't use this mysefl so it is not something I will implement at least, but contributions welcome!

**Comment by falematte** [comment]
Yep! Org babel is a great set of functions/tools that have been developed in the past years. They did a lot of work defining what is literate programming and creating an implementation of it. This can be a guide for some new features of this project.
